### DISCUSSION TITLE
Custom subject/body on hubs-cloud not changing the email

### DISCUSSION THREAD

**Discussion by camelman90** [mozilla/hubs#3417] [Hubs Cloud Questions] [answered]
I have set the fields for custom subject and body in the auth tab on app settings. no change to the subject or body of sign-in email being sent.

Am I missing something?

I have included the {{ link }} and tried just the link with some text, but it doesnt update even after letting it run for a couple of days still no change in the email being received.

No custom SMTP settings.

**Comment by robinkwilson** [comment, marked as answer]
This feature hasn't been pushed out to Hubs Cloud yet -- so that makes sense!

Do you have a custom client +  using master/main branch on Hubs? You need the backend update with the next HC auto update to make this work.

**Reply by camelman90** [reply]
That makes sense, i was thinking the admin panel was pushed seperatley but I am mostly on hubs/master which makes sense why its in admin but not on reticulum.  When I see "whats new" i should double check if its on hubs cloud or not in the future 😄  Thanks for answering!
